src/set_theory/game/nim.lean

```diff
@@ -69,22 +69,21 @@ by { rw nim_def, exact α.unique }
 noncomputable instance : unique (right_moves (nim 1)) :=
 by { rw nim_def, exact α.unique }
 
-/-- `0` has exactly the same moves as `nim 0`. -/
-def nim_zero_relabelling : relabelling 0 (nim 0) :=
-(relabelling.is_empty _).symm
+/-- `nim 0` has exactly the same moves as `0`. -/
+def nim_zero_relabelling : relabelling (nim 0) 0 := relabelling.is_empty _
 
-@[simp] theorem nim_zero_equiv : 0 ≈ nim 0 := nim_zero_relabelling.equiv
+@[simp] theorem nim_zero_equiv : nim 0 ≈ 0 := equiv.is_empty _
 
-/-- `nim 1` has exactly the same moves as `star`. -/
-noncomputable def nim_one_relabelling : relabelling star (nim 1) :=
+/-- `star` has exactly the same moves as `nim 1`. -/
+noncomputable def star_relabelling_nim : relabelling star (nim 1) :=
 begin
   rw nim_def,
   refine ⟨_, _, λ i,  _, λ j, _⟩,
   any_goals { dsimp, apply equiv_of_unique_of_unique },
-  all_goals { simp, exact nim_zero_relabelling }
+  all_goals { simp, exact nim_zero_relabelling.symm }
 end
 
-@[simp] theorem nim_one_equiv : star ≈ nim 1 := nim_one_relabelling.equiv
+@[simp] theorem star_equiv_nim : star ≈ nim 1 := star_relabelling_nim.equiv
```

Comment:
Does it make sense to flip this one around too? It feels like either you want `nim x` to be the simp normal form, or for `star` / `1` to be.

Reply:
I think there's a very slight reference to `star` here, only because it's a game that already has a name. I'll flip this lemma too.